Admin-Portal/src/pages/CashDonationsPage.tsx

```diff
@@ -321,7 +321,7 @@ const CashDonationsPage: React.FC = () => {
       {error && <ErrorMessage error={error} setError={setError} />}
       {success && <SuccessMessage success={success} setSuccess={setSuccess} />}
       <div style={{ display: "flex " }}>
-        <Calendar setFilterModel={setFilterModel} />
+        <Calendar handleFilterModelChange={handleFilterModelChange} />
```

Comment:
Is there a reason why you are not passing the rest two CalendarProps? It's giving errors in the Donation and CashDonation pages

<img width="537" alt="image" src="https://github.com/ChangePlusPlusVandy/MotherToMother/assets/81684017/290be648-8a09-4ead-923d-a53cca3d01f2">

Reply:
Oh I forgot. I will add those